conan/test_package/test_package.cpp

```diff
@@ -0,0 +1,68 @@
+#include "behaviortree_cpp/behavior_tree.h"
+#include "behaviortree_cpp/bt_factory.h"
+
+using namespace BT;
+
+NodeStatus SayHello()
+{
+    printf("hello\n");
+    return NodeStatus::SUCCESS;
+}
+
+class ActionTestNode : public ActionNode
+{
+  public:
+    ActionTestNode(const std::string& name) : ActionNode(name)
+    {
+    }
+
+    NodeStatus tick() override
+    {
+        time_ = 5;
+        stop_loop_ = false;
+        int i = 0;
+        while (!stop_loop_ && i++ < time_)
+        {
+            std::this_thread::sleep_for(std::chrono::milliseconds(100));
+        }
+        return NodeStatus::SUCCESS;
+    }
+
+    virtual void halt() override
+    {
+        stop_loop_ = true;
+        setStatus(NodeStatus::IDLE);
+    }
+
+  private:
+    int time_;
+    std::atomic_bool stop_loop_;
+};
+
+int main()
+{
+    BT::SequenceNode root("root");
```

Comment:
you wrote using namespace BT, so explicit namespace can be omitted here

Reply:
it's from https://github.com/BehaviorTree/BehaviorTree.CPP/blob/master/examples/broken_sequence.cpp